src/components/SideNavItem.tsx

```diff
@@ -23,27 +48,9 @@ class SideNavItem extends Component<Props> {
     } = this.props;
 
     return (
-      <Link
+      <StyledLink
         to={to}
-        css={[
-          {
-            display: 'flex',
-            alignItems: 'center',
-            justifyContent: 'space-between',
-            height: '3.2rem',
-            paddingLeft: level + '.2rem',
```

Comment:
There is em, rem and pixes used in site. Should we just use one type?

Reply:
Would be cleaner, but I'd leave this for the possible future rebuild of the site.